### PR TITLE
5382 - Ability to add axis labels in Bar Component

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR adds the ability to set axis labels (top, right, bottom, left) in the Bar Chart Component. The fix also includes the `RTL` position.

**Related github/jira issue (required)**:

Closes https://github.com/infor-design/enterprise/issues/5382

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.) LTR Position
- Go to http://localhost:4000/components/bar/example-axis-labels.html
- It should show labels on different positions (top, right, bottom, left) without any alignment issues
- Test in Classic Theme

2.) RTL Position
- Go to http://localhost:4000/components/bar/example-axis-labels.html?theme=new&mode=light&locale=ar-EG
- It should show labels on different positions (top, right, bottom, left) without any alignment issues
- Test in Classic Theme

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by yohannahbautista** [comment]
Hi, when you open the page in Edge in RTL, the numbers' positions are off. Though checking it here https://main-enterprise.demo.design.infor.com/components/bar/example-index.html?locale=ar-EG it doesn't look to be related to the labels
Screen Shot 2022-05-25 at 4 37 32 PM

**Comment by ericangeles** [comment]
@yohannahbautista looks like a bug on the previous version too. I'll fix this one too. Good catch!

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@
 
 ## v4.64.0 features
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:36]:
```suggestion
## v4.64.0 Features
```

**Review by tmcconechy — APPROVED** [review]

**Review by tjamesallen15 — APPROVED** [review]
Nice one!

**Review by yohannahbautista — APPROVED** [review]

**Pull request merged**